src/schema/resolvers.js

```diff
@@ -122,21 +129,32 @@ module.exports = {
       return await cursor.toArray();
     },
 
-    allOracles: async (root, {filter, first, skip, orderBy}, {mongo: {Oracles}}) => {
-      let query = filter ? {$or: buildOracleFilters(filter)}: {};
-      const cursor = Oracles.find(query);
-      if (first) {
-        cursor.limit(first);
+    allOracles: async(root, { filter, orderBy, limit, skip }, { mongo: { Oracles } }) => {
+      let query = filter ? { $or: buildOracleFilters(filter) } : {};
+
+      const options = {};
+
+      if (limit) {
+        options.limit(limit);
       }
 
       if (skip) {
-        cursor.skip(skip);
+        options.skip(skip);
       }
-      return await cursor.toArray();
+
+      if (!_.isEmpty(orderBy)) {
+        options.sort = [];
+
+        _.each(orderBy, (order) => {
+          options.sort.push([order.field, (order.direction === "ASC" ? 'asc' : 'desc')]);
```

Comment:
I notice direction can be null, so ensure the default order u want is `desc`. The mongo default is `asc`

Reply:
good point

Reply:
Yes current default is 'desc'.

Reply:
@chrisli30 sg, probably add a comment here